cmd/gitter/main.go

```diff
@@ -85,7 +85,7 @@ func gitter(repos []string, r *redis.Client) error {
 
 	for _, repoURL := range repos {
 		log.Printf("Indexing repo %s...\n", repoURL)
-		dir, err := ioutil.TempDir("tmp", "doc-gitter")
+		dir, err := ioutil.TempDir(os.TempDir(), "doc-gitter")
```

Comment:
This call was very unhappy in my env without this change. Worked in docker however.

Reply:
Yeah there is some funky logic here I need to correct, I'll open a PR and tag you on it :+1: